src/components/Books.js

```diff
@@ -1,13 +1,30 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
-import BookList from './BookList';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchBooks } from '../redux/books/books';
+import Book from './Book';
 import AddNewBook from './AddNewBook';
 
 const Books = () => {
-  const books = useSelector((state) => state.book);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchBooks());
+    // eslint-disable-next-line
+  }, []);
```

Comment:
- [ ] [OPTIONAL] Kindly make sure you fix your linter error instead of disenabling it 🙏